### ISSUE TITLE
Invisible blocks

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#153]
On the search screen while using Voiceover/Talkback there are invisible elements which cannot be identified by the screen reader.

All invisible elements should be removed and make sure the flow is as smooth as possible.

For the empty state illustrations: they should not be recognized, only the text and button should be labelled.

**Comment by ForumG-SC** [comment]
@Rajat-Sen Could you provide further details on the invisible blocks or if you could share the screen recording ? Thanks

**Comment by Rajat-Sen** [comment]
The issue only exists on the android phones. Please find the attached image for reference. Thanks
!
[Screenshot_20211231-132203](https://user-images.githubusercontent.com/38566088/147835620-7ae3af69-0234-46ba-8db1-320ab95d7f27.png)
!

**Comment by ForumG-SC** [comment]
@Rajat-Sen Can you share screen recording because I tested on android phone with the latest version I was not able to see the invisible block that you shared in the screenshot ?

**Comment by ForumG-SC** [comment]
![Screenshot_20211231-155953 (1)](https://user-images.githubusercontent.com/84411852/147838951-6fb369a1-5af3-4c86-9f94-36ca8370a2cb.jpg)
![Screenshot_20211231-155914 (1)](https://user-images.githubusercontent.com/84411852/147838953-83c6f9f9-28ed-4665-a35d-e9ad94e38f52.jpg)

This blocks seems to be normal behavior of how talk back works because as shown in the screenshot these blocks were there in all the applications. However, in search screen of StatsCAN app I don't see any block.

@Rajat-Sen Can you share screen recording because I tested on android phone with the latest version I was not able to see the invisible block on Search screen like the screenshot you shared?

**Comment by Rajat-Sen** [comment]
@ForumG-SC This issue is okay for a smaller invisible block. However, if there are recent searches, it focuses on a big invisible block. Please refer to the attached image.

**Comment by ForumG-SC** [comment]
@Rajat-Sen This issue is on Recent searches screen like you mentioned in the comment, I will look into it. Thanks.

**Comment by Rajat-Sen** [comment]
Also, it seems to exist only on Android devices.

I could not replicate this issue on iOS.

**Comment by ForumG-SC** [comment]
@AJBouzidi I don't have working device currently for this issue. Accessibility on my emulator is not responding.

**Comment by payeisa** [comment]
@AJBouzidi What's the status of this ticket?

**Comment by zhuelle** [comment]
@payeisa need prioritization.

**Comment by payeisa** [comment]
@zhuelle Priority (Minor) added for now, based there are a lot of other major priorities, and this issue target a small population.

**Comment by Rajat-Sen** [comment]
Closing this ticket for now as we are going to change it.
